ios/keyman/Keyman/Keyman/DropDownList/DropDownView.swift

```diff
@@ -0,0 +1,133 @@
+        tangent2End: CGPoint(x: currentFrame.width - strokeWidth, y: currentFrame.height - strokeWidth),
+        radius: borderRadius - strokeWidth)
+    context.addArc(tangent1End: CGPoint(x: currentFrame.width - strokeWidth,
+            y: currentFrame.height - strokeWidth),
+        tangent2End: CGPoint(x: round(currentFrame.width / 2.0 + arrowWidth / 2.0) - strokeWidth,
+            y: currentFrame.height - strokeWidth),
+        radius: borderRadius - strokeWidth)
+    context.addArc(tangent1End: CGPoint(x: strokeWidth, y: currentFrame.height - strokeWidth),
+        tangent2End: CGPoint(x: strokeWidth, y: arrowPosY),
+        radius: borderRadius - strokeWidth)
+    context.addArc(tangent1End: CGPoint(x: strokeWidth, y: arrowPosY),
+        tangent2End: CGPoint(x: currentFrame.width - strokeWidth, y: arrowPosY),
+        radius: borderRadius - strokeWidth)
+    context.closePath()
+    context.drawPath(using: CGPathDrawingMode.fillStroke)
+
+    context.beginPath()
+    context.move(to: CGPoint(x: borderRadius + strokeWidth, y: arrowPosY))
+    context.addLine(to: CGPoint(x: arrowPosX - arrowWidth / 2.0, y: arrowPosY))
+    context.addLine(to: CGPoint(x: arrowPosX, y: strokeWidth))
+    context.addLine(to: CGPoint(x: arrowPosX + arrowWidth / 2.0, y: arrowPosY))
+    context.addArc(tangent1End: CGPoint(x: currentFrame.width - strokeWidth, y: arrowPosY),
+        tangent2End: CGPoint(x: currentFrame.width - strokeWidth, y: currentFrame.height - strokeWidth),
+        radius: borderRadius - strokeWidth)
+    context.addArc(tangent1End: CGPoint(x: currentFrame.width - strokeWidth,
+            y: currentFrame.height - strokeWidth),
+        tangent2End: CGPoint(x: round(currentFrame.width / 2.0 + arrowWidth / 2.0) - strokeWidth,
+            y: currentFrame.height - strokeWidth),
+        radius: borderRadius - strokeWidth)
+    context.addArc(tangent1End: CGPoint(x: strokeWidth, y: currentFrame.height - strokeWidth),
+        tangent2End: CGPoint(x: strokeWidth, y: arrowPosY),
+        radius: borderRadius - strokeWidth)
+    context.addArc(tangent1End: CGPoint(x: strokeWidth, y: arrowPosY),
+        tangent2End: CGPoint(x: currentFrame.width - strokeWidth, y: arrowPosY),
+        radius: borderRadius - strokeWidth)
+    context.closePath()
+    context.clip()
+
+    let colorSpace = CGColorSpaceCreateDeviceRGB()
+    let gradientColors = [bgColor.cgColor, backgroundColor2.cgColor] as CFArray
+    let gradientLocations: [CGFloat] = [0, 1]
+    guard let gradient = CGGradient(colorsSpace: colorSpace, colors: gradientColors,
+        locations: gradientLocations) else {
+      return
+    }
+
+    let startPoint = CGPoint(x: rect.midX, y: rect.minY)
+    let endPoint = CGPoint(x: rect.midX, y: rect.maxY)
+
+    context.drawLinearGradient(gradient, start: startPoint, end: endPoint, options: [])
+    UIGraphicsPopContext()
+  }
+
+  override var backgroundColor: UIColor? {
```

Comment:
What happened to `setBackgroundColor2`?  It appears that this property got lost in the language-translation process.

Reply:
https://github.com/keymanapp/keyman/blob/136e08b2f1c66ba856443ec9c976ba3e7c22cc71/ios/keyman/Keyman/Keyman/DropDownList/DropDownView.swift#L20

`backgroundColor2` is a stored property. When bridging to Objective C `setBackgroundColor2()` is generated so we don't need an explicit setter.